qualtran/bloqs/rotations/hamming_weight_phasing_test.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import cirq
+import numpy as np
+import pytest
+
+from qualtran.bloqs.rotations.hamming_weight_phasing import HammingWeightPhasing
+from qualtran.cirq_interop.testing import GateHelper
+from qualtran.testing import assert_valid_bloq_decomposition
+
+
+@pytest.mark.parametrize('n', [2, 3, 4, 5, 6, 7, 8])
+@pytest.mark.parametrize('theta', [1 / 10, 1 / 5, 1 / 7, np.pi / 2])
+def test_hamming_weight_phasing(n: int, theta: float):
+    gate = HammingWeightPhasing(n, theta)
+    assert_valid_bloq_decomposition(gate)
+
+    print(gate.t_complexity())
```

Comment:
remove print

Reply:
bump

Reply:
This is gone? 

![Uploading image.png…]()

Reply:
Oh weird, sorry my browser didn't update.